components/rmrk/Gallery/GalleryCard.vue

```diff
@@ -14,11 +14,10 @@
           <span class="card-image__emotes__count">{{ emoteCount }}</span>
         </span>
         <BasicImage
-          v-if="!animatedUrl"
+          v-if="!animatedUrl || image"
```

Comment:
I'm afraid that some 3d models nft could not be shown properly because it has `image`.

Could you pls explain why ?

Reply:
let me check other 3d collections again, or do you have which 3d items have `animatedUrl` and `image` at the same time?

afaik for 3d use `animatedUrl` only

Reply:
initially, this is for handling `audio` items because it has `animatedUrl` and `image`. so, instead of showing the audio player on the collection page, `audio` items show the image instead

Reply:
tested in another 3d collections seems fine [`34db249` (#3799)](https://github.com/kodadot/nft-gallery/pull/3799/commits/34db24931b3a9c92ec8930ce19a69015ff518906#diff-8917895a9cfaf52ac3c6af17865d77851ad60ab4401d2b1581e7776b76abe14d)

Reply:
Oki